40-car-detector/main.py

```diff
@@ -0,0 +1,227 @@
+import sys
+from machine import I2C, Pin, RTC
+import sensor
+import time
+import tf
+import math
+from collections import namedtuple
+
+# Add /lib to sys.path so that we can import notecard.
+sys.path.append('/lib')
+import notecard
+
+Config = namedtuple('Config', 'confidence_threshold publish_period')
+
+
+def setup_notecard(product_uid):
+    # Setup I2C channel for communicating with the Notecard.
+    i2c = I2C(2)
+    card = notecard.OpenI2C(i2c, notecard.NOTECARD_I2C_ADDRESS, 0, debug=True)
+
+    req = {
+        'req': 'hub.set',
+        'product': product_uid,
+        'mode': 'periodic',
+        'outbound': 5
+    }
+    card.Transaction(req)
+
+    # Establish a template Notefile, events.qo, where we'll push car events.
+    req = {
+        'req': 'note.template',
+        'file': 'events.qo',
+        'body': {
+            'start': 24,
```

Comment:
It's a pity we don't have the template constants defined in note-python. It might be worth adding a link to the note.template docs where these are listed.